### ISSUE TITLE
database & local storage parts in Readme

### ISSUE THREAD

**Issue by jancborchardt** [agical/friendsunhosted#11]
What are the steps »Install redis database« and »Get a local storage (express-storage)« exactly for? It’s a bit confusing since it seems that the redis database is needed for the app itself.

If they are for running your own remoteStorage locally, I’d say we should instead recommend people to use their proper remoteStorage. If they want to install it locally for testing, we can refer to the express-storage readme.

**Comment by brolund** [comment]
Yes, they are only for running the tests locally (which was the main reason for the README :-)).

Since express-storage pulls in all sorts of persona/mozilla stuff, including some login popup (which was not in the version I started with), I patched express storage to work better with the tests. This is not optimal, but the best way I could think of.
